app/src/main/java/org/p2p/wallet/deeplinks/AppDeeplinksManager.kt

```diff
@@ -49,11 +54,17 @@ class AppDeeplinksManager(
 
     fun handleDeeplinkIntent(intent: Intent) {
         when {
-            isAppOpenFromBackground(intent) -> {
+            isDeeplinkWithUri(intent) -> {
                 val data = intent.data ?: return
-                intercomDeeplinkManager.handleBackgroundDeeplink(data)
+                if (context.getString(R.string.app_scheme) == data.scheme &&
+                    data.host in DeeplinkHosts.validListToStartDeeplinks
```

Comment:
does it fit in 1 line? 
If not, I propose to move `data.host in DeeplinkHosts.validListToStartDeeplinks` to a separate boolean value

Reply:
no i line fail ktlint
ok will move to a separate boolean value